### ISSUE TITLE
model_segnet_mtan prediction code

### ISSUE THREAD

**Issue by AHV137335** [lorenmt/mtan#31]
Hello and sorry to bother.
I am currently trying to train a multitask segnet model on NYUv2 dataset. I just have a few questions about it and hope you could help me.
1) After running the code and training for long enough, where the trained model will save? The wide resnet codes have torch.save at the bottom lines of the script. but the segnet models don't include the same lines and basically after the system finishes running the script I can't find any model as the result. Of course results of the evaluation on all 3 tasks will be given to me but I just can't find the saved model.

2) I also wanted to ask for your help about the prediction code. Is there a way to give the trained model and one desired image to some prediction script and get 3 different image outputs for semantic masks and depth plus the surface normalization?

Best regards.

**Comment by lorenmt** [comment]
Hi,

Thanks for being interested in my work.

1. The major reason I did not include the model saving code in NYUv2 is simply because that the dataset is quite small, and so it won't take a long time to converge.

2. Any evaluation stage of the code does that, which is to take an unseen test image to a trained network and evaluate its respective performance.

The code is mainly to help the readers better understand the terminologies and design in the original paper. You may revise as much as you can to fit your own preference. For any other general questions on PyTorch evaluation and coding technique, I suggest taking a look at some tutorial page at the official PyTorch website, which should be helpful.

**Comment by AHV137335** [comment]
Thanks for your quick answer.
